### DISCUSSION TITLE
Übersetzungsvorschlag Mining 3.23.1 Live

### DISCUSSION THREAD

**Discussion by BTSVDanielk** [rjcncpt/StarCitizen-Deutsch-INI#543] [Übersetzungsvorschläge]
Hallo Übersetzungsteam,

danke für Eure tägliche Arbeit für die Star Citizen Deutsch Übersetzung!

Ich habe heute mit meiner Prospector einen Mining Run gestartet und mir ist aufgefallen, dass neue Übersetzungen im Mining Bereich sind.

Die Wörter gefallen mir, aber man kann sie nicht ausgeschrieben lesen oder sie überdecken die SCU Zahl in der Fracht- Anzeige, deshalb habe ich einen Vorschlag für die Mining Scan Ergebnisse und die Fracht Anzeige siehe Bild Rot markiertes Quadrat.

Bei Hephaestanite steht rechts daneben unraffiniert, dieses Wort könnte man mit Erz ergänzen, wie z.B Gold und Alluminium "siehe rotes Quadrat.

Ich weiss aber nicht ob Hephaestanite auch ein Erz ist, deshalb könnte man es auch einfach Mineral nennen. Oder etwas anderes, sodass zumindest die gesammelte SCU Zahl in der Fracht- Anzeige angezeigt wird.

Das Wort nicht wertvolle Materialien überdeckt auch die gesammelte SCU Zahl in der Frachtanzeige.
Hierfür würde mir auf die schnelle nur wertlose Materialien einfallen.

Ich habe Urlaub und werde diese Woche bestimmt noch öfters das Mining Gamplay spielen und falls mir noch ein anderes Wort auffält, dass die SCU Anzeige überdeckt, werde ich mich hier unter diesen Posts nocheinmal melden.

Danke fürs Durchlesen und nochmal danke für die Übersetzung.

Gruß Daniel K

![11](https://github.com/rjcncpt/StarCitizen-Deutsch-INI/assets/167604723/05b872b9-59d7-4f0a-a73b-157be8b751a1)

**Comment by rjcncpt** [comment]
Hey Daniel, Danke für deine Vorschläge. Ich habe "unraffiniert" zu "Roh" geändert, weil es im englischen auch "Raw" geschrieben steht und "Nicht wertvolle Materialien" zu "Wertloses Material" geändert.

Ich denke so sollte es dann passen :) Du kannst dir die global.ini jetzt herunterladen: https://github.com/rjcncpt/StarCitizen-Deutsch-INI/blob/main/live/full/global.ini

Lieben Gruß
René

**Comment by BTSVDanielk** [comment]
Danke für das ändern. Die neue Datei habe ich bei mir erfolgreich in mein Verzeichnis kopiert. Wenn ich irgendwann etwas finde, melde ich mich wieder.

**Reply by rjcncpt** [reply]
Vielen Dank :)

**Reply by BTSVDanielk** [reply]
Der Bestätigungs Button um Fracht in ein Raumschiff zu laden, dort fehlt ein N. Ich weiß nicht ob der Buchstabe  vergessen wurde oder das Feld zu kurz für unsere deutsche Übersetzung ist. Übertragen hört sich eigentlich gut an aber falls das Feld zu kurz ist könnte man liefern oder zuweisen nehmen. 

![ScreenShot-2024-05-22_23-31-05-1B6](https://github.com/rjcncpt/StarCitizen-Deutsch-INI/assets/167604723/2372f638-9f9e-4bda-8800-04e74e0eea3a)

**Reply by BTSVDanielk** [reply]
Wenn die Masse eines gebrochenen Steines kleiner als 1 SCU ist dann überlappt sich die SCU Zahl mit dem Wort Zusammensetzung. Ich würde da Masse hinschreiben weil das Wort sich auch auf die SCU Zahl beziehen könnte, ich weiß nicht was im Englischen steht. Oder ein anderes Wort. :) Wenn dies ersmal nicht geändert wird, ist es wahrscheinlich auch garnicht so schlimm, vielen würde das vielleicht garnicht auffallen *g Gruß Daniel

![ScreenShot-2024-05-21_07-14-19-62B](https://github.com/rjcncpt/StarCitizen-Deutsch-INI/assets/167604723/4c42acc7-76df-4430-a5df-25d701f6437d)

**Reply by rjcncpt** [reply]
Hey Daniel. 

Also dort steht im englichen "Composition". Ich glaube Masse passt da nicht so ganz. Ich würde "Aufbau" nehmen?

**Reply by rjcncpt** [reply]
> Der Bestätigungs Button um Fracht in ein Raumschiff zu laden, dort fehlt ein N. Ich weiß nicht ob der Buchstabe vergessen wurde oder das Feld zu kurz für unsere deutsche Übersetzung ist. Übertragen hört sich eigentlich gut an aber falls das Feld zu kurz ist könnte man liefern oder zuweisen nehmen.
> 
> ![ScreenShot-2024-05-22_23-31-05-1B6](https://private-user-images.githubusercontent.com/167604723/332969915-2372f638-9f9e-4bda-8800-04e74e0eea3a.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY1NzY3MDYsIm5iZiI6MTcxNjU3NjQwNiwicGF0aCI6Ii8xNjc2MDQ3MjMvMzMyOTY5OTE1LTIzNzJmNjM4LTlmOWUtNGJkYS04ODAwLTA0ZTc0ZTBlZWEzYS5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUyNFQxODQ2NDZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iNzNkM2QyNThkZTBhNDU5NjVkMmVlMjk5MzUwNDM2Y2I2ZTdjMWNjZTAxYTBkZTQxNzM5NTM1ZTBkOGVkOGMxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.9QbbKXqQRxWahTdxZ25kz31XeTO60WVYLHukmSz4Hq4)

Könntest du das bitte mal testen? Ich komme aktuell nicht dazu ins Spiel zu gehen. Ändere in der ini Datei mal folgende Zeilen um und schreib mir welche Nummer an dieser Stelle erscheint.

Usable_Terminal_Status_Extracting=1Übertragen
Usable_Terminal_Withdraw=2Übertragen

**Reply by BTSVDanielk** [reply]
Aufbau hört sich gut an. Ist sogar viel kürzer als das jetzige Wort.
Beim zweiten Post kann ich leider nicht Helfen, ich habe nix gefunden und wüßte nicht so wo ich nachsehen soll. Ich kenn mich damit nicht so gut aus ;)

**Reply by rjcncpt** [reply]
Du brauchst nur die global.ini Datei mit einem Texteditor öffnen und die Zeilen jeweils

Usable_Terminal_Status_Extracting=
Usable_Terminal_Withdraw=

in die Suche eingeben und dann wie bei meinem Beispiel oben nummerieren und speichern. Dann siehst du das so im Spiel.

😊

**Reply by BTSVDanielk** [reply]
Ich habe die Zeilen in der global.ini gefunden und in 

Usable_Terminal_Status_Extracting=1Übertragen
Usable_Terminal_Withdraw=2Übertragen

umgeändert. Im Spiel habe ich aber keine Ändernugen gesehen. Wo sollte es eine Änderung geben?

Ich habe aber etwas neues gefunden was nicht ganz perfekt ist. Der Rote schrifzug Fracht Voll überlappt sich mit der Gesamt SCU Anzeige, ich habe zwar 48 SCU weil ich die Mole Behälter an meiner Prospector habe. Aber bei der normalen SCU Anzeige der Prospector von 32 SCU werden sich die beiden Wort/Zahl auch in die quere kommen.

![ScreenShot-2024-05-25_10-22-25-ADB](https://github.com/rjcncpt/StarCitizen-Deutsch-INI/assets/167604723/3b453555-d852-4d77-bf51-2b426dcc0e5d)

**Comment by rjcncpt** [comment]
Oh, vorzeitig geschlossen. Ich habe "Fracht voll" einfach durch "Voll" ersetzt. Kommt mit dem nächsten Update :)

**Reply by BTSVDanielk** [reply]
Danke
